Kconfig

```diff
@@ -26,7 +26,7 @@ if FOTA_DEVICE_SOC_SERIES_KINETIS_K6X
 
 # IPv6 is enabled by default in Zephyr's Kconfig when networking is selected.
 config NET_IPV6
-	default n
+	default n if FOTA_DEVICE_SOC_SERIES_KINETIS_K6X
```

Comment:
Woops.  Just realized this is already in side a FOTA_DEVICE_SOC_SERIES_KINETIS_K6X block